test/channels/application_cable/connection_test.rb

```diff
@@ -1,4 +1,4 @@
-require "test_helper"
+require 'test_helper'
```

Comment:
- [ ] Please make sure App has unit tests for all the most important components of the RoR application.

Reply:
All important unit tests were written and all are passing
![rspec-pass](https://user-images.githubusercontent.com/92439670/204101250-8e196f75-65ac-42a8-a909-dc68ddaaa1d9.PNG)